### PR TITLE
Update dependencies

### PR DESCRIPTION
**Pull request by beorn7**
In preparation for the next release.

This also bumps the minimum required Go version to 1.13, which is required by the prometheus/procfs package using the new
`%w` printf directives. However, it also allows us to remove some special casing about build info.

This also adds 1.16 to the testing. Thus, we still support the last four minor releases, which should be plenty.

### CODE REVIEW

**Comment by beorn7** [comment]
Not sure why the old test still show up. They have been removed from the circleci config.

**Comment by beorn7** [comment]
Perhaps because only the 2nd commit removed them?

**Comment by beorn7** [comment]
Ping?

**Review by SuperQ — APPROVED** [review]
LGTM

**Comment by SuperQ** [comment]
I updated the required CI tests.

File: .circleci/config.yml [hunk 1]
```diff
@@ -78,6 +62,10 @@ workflows:
         name: go-1-15
         go_version: "1.15"
         run_lint: true
```
Comment (roidelapluie) [inline .circleci/config.yml:64]:
```suggestion
        run_lint: false
```

**Review by roidelapluie — APPROVED** [review]

**Comment by beorn7** [comment]
@SuperQ :
> I updated the required CI tests.

What did you do?

**Comment by roidelapluie** [comment]
In github you can mark tests as `required`, in which case they will always be present in the UI and must pass. They would appear even if they disappear in circleci.

**Comment by beorn7** [comment]
I had a vague recollection of that setting, but I could not find it anywhere. Am I too stupid for GH's level of UX?

**Comment by beorn7** [comment]
OK, found it. It's a branch protection rule...

**Comment by roidelapluie** [comment]
Settings > Branches > Branch Protection rules > master - Edit > Require status checks to pass before merging

**Pull request merged**